CONTRIBUTING.md

```diff
@@ -270,15 +220,13 @@ it SHOULD receive one if that version is a patch version.
 
 #### Creating the new stable branch and files
 
-If the current major is `42`, a new `43.x` branch SHOULD be created from master,
-then a commit SHOULD be done on master to bump the `branch-alias` and version numbers in the README.
+If the current major is `42`, a new `43.x` branch SHOULD be created from `42.x`.
 
 Also, the following files MUST be created/updated on the new stable branch:
 
  - `UPGRADE-43.x.md`, containing only the main title
  - `UPGRADE-43.0.md`, containing the upgrade notes fetched from the major PRs.
  - `CHANGELOG.md`, containing the changelog of the major PRs.
- - `composer.json`, the `branch-alias` MUST be changed to `43.x-dev`
 
 Push the new branch with a commit containing the modified files and "43.x-dev" as comment.
```

Comment:
I think this line should change or be removed too

Reply:
Done